include/gmlWriter.h

```diff
@@ -0,0 +1,28 @@
+#ifndef GMLWRITER_H
+#define GMLWRITER_H
+
+#include <QXmlStreamWriter>
+#include <QFile>
+#include <typedefs.h>
+
+#include "transmem.h"
+
+class TransMem;
+
+class GMLWriter {
```

Comment:
GraphML is a XML kind of format to represent graphs: https://en.wikipedia.org/wiki/GraphML

One can for example use Gephi to display a GraphML file: https://gephi.org/

I added a description and also changed the name of the class to GraphMLWriter, what is more correct.